sessions/react-global-state/demo-end/pages/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+const Title = styled.h1`
+  font-size: 1rem;
+  margin: 0;
+`;
+
+export default function AppHeader({ countSum }) {
```

Comment:
```suggestion
export default function Header({ countSum }) {
```

Name the function like the file :)